### DISCUSSION TITLE
So I love being angry so I have an idea of a new fresh hell to install for the game and I think it would spice up things AND would "Probably" not be hard to code..

### DISCUSSION THREAD

**Discussion by ExLegendary** [FakeFishGames/Barotrauma#9008] [Ideas and suggestions]
So...Energy Vampires....

I would like to have small little enemies that move fast and stick to the ship and drain its power....You physically have to climb out of the ship to shoot them but don't do hull damage. Make them swarm with other enemies so while you fight other monsters you have to deal with your fuel being drained far faster and potentially your electrical shorting out... 

Next I would want a giant enemy that can harpoon the ship *Moloch sized" that can drain the ship far faster than the biters and attacks the ship...My idea is the harpoon has to be physically cut off from inside the hull to stop it from draining your ships power and damaging the electrical.

I think too many enemy types focus on the pure physical danger I want someone that makes it a complete threat to your ship and crews ability to keep the ship running in a whole new area!

Think like Remoras with energy vampirism and like say a subtle nod to James and the Giant Peaches robot shark or something...

**Comment by ghost** [comment]
arcfiends and giant tentacle monsters. seems legit.
